PdTest/build.gradle

```diff
@@ -6,12 +6,12 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 21
-    buildToolsVersion "21.1.2"
+    compileSdkVersion 23
+    buildToolsVersion "23.0.1"
 
     defaultConfig {
         minSdkVersion 10
-        targetSdkVersion 21
+        targetSdkVersion 23
```

Comment:
API 23 introduces permission changes for apps, and PdTest manifest requests two permissions. Even though this is a test app, I'm concerned that targeting API 23 should be accompanied by changes in permission handling.

http://developer.android.com/training/permissions/requesting.html